files/README.md

```diff
@@ -15,3 +15,28 @@ Then compile and run the service:
 go build
 ./files
 ```
+
+## Resources
+
+This example contains following two resources.
+
+* schema
+* swagger
+
+### Schema
+
+Schema is served from native file system so you have to deploy the binary with `public/schema` directory and keep their path relationship.
```

Comment:
The `schema` resource contains a single endpoint which serves the static files contained in the `public/schema` directory. This means the directory must exist and its relative path must be `public/schema`.